TransIT.Tests/Repository/EmployeeRepositoryTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using TransIT.DAL.Models;
+using TransIT.DAL.Models.Entities;
+using TransIT.DAL.Repositories.ImplementedRepositories;
+using Xunit;
+
+namespace TransIT.Tests
+{
+    public class EmployeeRepositoryTests
+    {
+        [Fact]
+        public async Task Employee_Repository_Should_Add_Employee()
+        {
+            var context = CreateDbContext();
```

Comment:
add Arrange, Act Asser comments to divide parts of test method(for all test methods)